core/dir.c

```diff
@@ -74,6 +74,21 @@ oio_directory__set_properties(struct oio_directory_s *self,
 	DIR_CALL(self, set_prop)(self, url, values);
 }
 
+GError *
+oio_directory__force (struct oio_directory_s *self,
```

Comment:
Please don't put a space between a function name and the open parenthesis. This is reserved for conditions (`if`, `for`, `while`).